apple-codesign/Cargo.toml

```
@@ -39,7 +41,10 @@ figment = { version = "0.10.13", features = ["env", "toml"] }
 filetime = "0.2.23"
 glob = "0.3.1"
 goblin = "0.8.0"
+headers = "0.3.8"
 hex = "0.4.3"
+hyper = { version = "0.14.27", features = ["client"] }
+hyper-proxy = "0.9.1"
```

`hyper-proxy` is adding a dependency on `openssl`, which is causing the CI failure and makes building more complicated.

We can probably avoid this dependency by telling hyper-proxy to use rustls instead:

```suggestion
hyper-proxy = { version = "0.9.1", default-features = false, features = ["rustls"] }
```